gammapy/data/metadata.py

```diff
@@ -0,0 +1,90 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from typing import Optional, Union
+from astropy.coordinates import EarthLocation, SkyCoord
+from astropy.time import Time
+from pydantic import Field, validator
+from gammapy.utils.fits import earth_location_from_dict
+from gammapy.utils.metadata import CreatorMetaData, MetaData
+
+__all__ = ["ObservationMetaData"]
+
+
+class ObservationMetaData(MetaData):
+    """Metadata containing information about the Observation.
+
+    Parameters
+    ----------
+    telescope : str, optional
+        the telescope/observatory name
+    instrument : str, optional
+        the specific instrument used
+    observation_mode : str, optional
+        the observation mode
+    location : `~astropy.coordinates.EarthLocation` or str, optional
+        the observatory location
+    deadtime : float
+        the observation deadtime. Default is 1.
```

Comment:
This default means that there is no observation time. Is it what we want? 0 is better
I think that no default is wished, this quantity should be filled all the time...

Reply:
Yes I followed the convention used in gadf  to store the deadtime correction factor. This might not be a good choice, especially if the attribute is called `deadtime`...

What do we prefer: storing the deadtime or the correction factor?

Reply:
Store livetime, start of observation and end?